books/RayTracingInOneWeekend.html

```diff
@@ -1864,34 +1896,23 @@
 
 </div>
 
-They also might be absorbed rather than reflected. The darker the surface, the more likely
-absorption is. (That’s why it is dark!) Really any algorithm that randomizes direction will produce
-surfaces that look matte. One of the simplest ways to do this turns out to be exactly correct for
-ideal diffuse surfaces. (I used to do it as a lazy hack that approximates mathematically ideal
-Lambertian.)
-
-(Reader Vassillen Chizhov proved that the lazy hack is indeed just a lazy hack and is inaccurate.
-The correct representation of ideal Lambertian isn't much more work, and is presented at the end of
-the chapter.)
-
 <div class='together'>
-There are two unit radius spheres tangent to the hit point $p$ of a surface. These two spheres have
-a center of $(\mathbf{P} + \mathbf{n})$ and $(\mathbf{P} - \mathbf{n})$, where $\mathbf{n}$ is the
-normal of the surface. The sphere with a center at $(\mathbf{P} - \mathbf{n})$ is considered
-_inside_ the surface, whereas the sphere with center $(\mathbf{P} + \mathbf{n})$ is considered
-_outside_ the surface. Select the tangent unit radius sphere that is on the same side of the surface
-as the ray origin. Pick a random point $\mathbf{S}$ inside this unit radius sphere and send a ray
-from the hit point $\mathbf{P}$ to the random point $\mathbf{S}$ (this is the vector
-$(\mathbf{S}-\mathbf{P})$):
+They might also be absorbed rather than reflected. The darker the surface, the more likely
+the ray is absorbed (that’s why it's dark!). Really any algorithm that randomizes direction will
+produce surfaces that look matte. Let's start with the most intuitive: a surface that randomly
+bounces a ray equally in all directions. For this material, any ray that hits the surface has an
+equal probability of bouncing off toward any direction above the horizon:
```

Comment:
"Horizon" is confusing here, as it typically means the world horizon. Maybe use a phrase like "any direction away from the surface"? ("Above" is also tricky.)

Reply:
This also caught my eye. I didn't really like any alternative to "horizon" but i'll noodle on it.

Reply:
I more or less did your suggestion exactly